docs/tutorials/evpn-mh/index.md

```diff
@@ -0,0 +1,191 @@
+| **Lab components**        | 4 SR Linux nodes, 2 Alpine Linux                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| **Resource requirements** | :fontawesome-solid-microchip: 3vCPU <br/>:fontawesome-solid-memory: 6 GB                                                                                                                                                                                                                                                                                                                                                                                             |  |
+| **Lab**                   | [srl-labs/srl-evpn-mh-lab][lab]                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| **Main ref documents**    | [RFC 7432 - BGP MPLS-Based Ethernet VPN](https://datatracker.ietf.org/doc/html/rfc7432)<br/>[RFC 8365 - A Network Virtualization Overlay Solution Using Ethernet VPN (EVPN)](https://datatracker.ietf.org/doc/html/rfc8365)<br/>[Nokia 7220 SR Linux Advanced Solutions Guide](https://documentation.nokia.com/srlinux/23-3/books/advanced-solutions)<br/>[Nokia 7220 SR Linux EVPN-VXLAN Guide](https://documentation.nokia.com/srlinux/23-7/title/evpn_vxlan.html) |
+| **Version information**   | [`containerlab:0.44.0`][clab-install], [`srlinux:23.7.1`][srlinux-container], [`docker-ce:23.0.3`][docker-install]                                                                                                                                                                                                                                                                                                                                                   |
+
+One of the many advantages of EVPN is its built-in multi-homing (MH) capability, which is standards-based and defined by RFCs 7432, 8365.
+
```

Comment:
@aaakpinar we need to use `multihoming` across the whole post, since this is the term used in rfc (meaning not with a dash multi-homing)

Reply:
done